baseplate/live_data/__init__.py

```diff
@@ -0,0 +1,11 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+#from __future__ import unicode_literals This breaks __all__ on PY2
```

Comment:
I do want to keep the comment so it's obvious why that `__future__` setting isn't being used here. Given that page, I agree that we should re-evaluate using that flag at all, but would like to keep that for a separate change.